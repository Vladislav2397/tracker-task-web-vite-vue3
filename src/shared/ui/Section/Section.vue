<template>
    <div data-testid="Section" class="section">
        <slot />
    </div>
</template>

<style scoped>
.section {
    padding: 12px;
    box-sizing: border-box;
    background-color: #000;
}
</style>
