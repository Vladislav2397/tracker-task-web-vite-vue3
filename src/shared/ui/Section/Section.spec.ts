import { render } from '@testing-library/vue'
import Section from './Section.vue'

describe('shared/ui/Section component', () => {
    it('Snapshot', () => {
        const { html } = render(Section)

        expect(html()).toMatchSnapshot()
    })
})
