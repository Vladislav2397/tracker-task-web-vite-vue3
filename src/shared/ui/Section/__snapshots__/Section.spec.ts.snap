// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`shared/ui/Section component > Snapshot 1`] = `"<div data-v-8736e483=\\"\\" data-testid=\\"Section\\" class=\\"section\\"></div>"`;
